src/framing/sender.rs

```diff
@@ -74,6 +74,9 @@ impl Sender {
     pub(crate) fn as_node_index(self) -> NodeIndex {
         NodeIndex::from(self.sender)
     }
+    pub fn as_leaf_index(self) -> LeafIndex {
```

Comment:
This has been bad in the name above as well, but these functions should be `to_leaf_index` (or `into_` as it's not really expensive). `as_` implies borrows.
But let's not do it in this PR.